src/Pages/Login/EditUserName.js

```diff
@@ -0,0 +1,217 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+import { GET_EDITINFO_API, GET_SIGNUP_API } from '../../config';
+
+function EditUserName() {
+  const [userInfo, setUserInfo] = useState(
+    JSON.parse(localStorage.getItem('user-info'))
+  );
+  const [userNickName, setUserNickName] = useState('');
+  const [caution, setCaution] = useState('');
+  const [isOverLap, setIsOverLap] = useState(false);
+  const [suggestName, setSuggestName] = useState('');
+  const history = useHistory();
+
+  const validationCheck = e => {
+    setUserNickName(e.target.value);
+  };
+
+  useEffect(() => {
+    setUserNickName(userInfo.nickname);
+  }, []);
+
+  useEffect(() => {
+    if (userNickName.length === 0) {
+      setCaution('필수 항목 입니다.');
+      setIsOverLap(false);
+    } else if (userNickName.length === 1) {
+      setCaution('2자 이상 입력해주세요');
+      setIsOverLap(false);
+    } else if (isOverLap) {
+      setCaution('사용 중인 닉네임입니다.');
+    } else {
+      setCaution('');
+    }
+  }, [userNickName, isOverLap]);
+
+  const checkId = () => {
+    if (userNickName.length > 1) {
+      console.log('패치!');
```

Comment:
```suggestion
    if (userNickName.length) {
```
0은 false임을 활용해서 좀 더 간략하게 표시할 수 있겠네요!
그리고 테스트가 끝난 콘솔은 지워주세요! :)